server/server.js

```diff
@@ -11,6 +12,17 @@ app.use((req, res, next) => {
 // serve static assets
 app.use(express.static(path.join(__dirname, '../client/dist')));
 
+// outputs all comments for a song
+app.get('/api/songs/songId/comments', (req, res) => {
```

Comment:
Are you trying to have a dynamic songId? If so, use
:songId

Reply:
It's static and a stand in since I don't handle song ID's.